specification/communication/data-plane/Chat/preview/2024-03-15-preview/communicationserviceschat.json

```
@@ -1229,7 +1229,7 @@
           {
             "in": "body",
             "name": "chatImageFile",
-            "description": "Read receipt details.",
+            "description": "Image binary data, allowed image formate: jpg/jpeg, png, gif, heic, webp",
```

```suggestion
            "description": "Image binary data, allowed image formats: jpeg, png, gif, heic, webp",
```

Misspelling, and "jpg" vs. "jpge" are not different formats - just different common extensions. I wouldn't list both. As you noted, the extension really doesn't matter anyway.